drivers/amazonec2/region.go

```diff
@@ -22,6 +22,7 @@ var regionDetails map[string]*region = map[string]*region{
 	"ap-south-2":      {"ami-0eaa3ccae04723740"},
 	"ap-east-1":       {"ami-0a7893426ebe74a8c"},
 	"ca-central-1":    {"ami-095509bf36d02a8e0"},
+	"ca-west-1":       {"ami-07cfb3e4b8d29eaf4"},
```

Comment:
```suggestion
	"ca-west-1":       {"ami-07ce2430f97a1fad1"},
```

The latest AMI for `Ubuntu 20.04 LTS hvm:ebs-ssd (amd64)` is `ami-07ce2430f97a1fad1` on https://cloud-images.ubuntu.com/locator/ec2/. 
Can you update your PR?

Reply:
done